lib/src/generator/dartdoc_generator_backend.dart

```diff
@@ -204,12 +202,26 @@ abstract class DartdocGeneratorBackend implements GeneratorBackend {
         sidebarForContainer.getRenderFor);
     var content = templates.renderMethod(data);
     write(writer, method.filePath, data, content);
+    runtimeStats.accumulators.update('writtenMethodFileCount', (c) => c + 1);
```

Comment:
We could add a method `incrementAccumulator` to `_RuntimeStats`, and maybe make the `Map accumulators` itself private.

Reply:
Done